### ISSUE TITLE
Summary highlight broken

### ISSUE THREAD

**Issue by MikaelMayer** [dafny-lang/ide-vscode#268]
When I see this on the gutter
![image](https://user-images.githubusercontent.com/3601079/190439342-01aec14a-e572-4924-99fc-d2beaf0d027b.png)

the summary on the scroll bar shows this.
![image](https://user-images.githubusercontent.com/3601079/190439433-3c0f9738-c48a-4a46-9f9c-b96df0cfc8ad.png)
whereas I would have expected everything to be green.

I understand we are not displaying the gutter under the test icons, but it should not affect the summary

**Comment by keyboardDrummer** [comment]
I think it would be better to turn off the continuous green line if the testing checkmarks are enabled, since both are trying to convey the same information but with a different UI.

**Comment by MikaelMayer** [comment]
I don't agree with this :-) Both can be very useful. For example, testing checkmarks can indicate if a module or a class has errors, which is not possible using the continuous gutter icons.
However, the continuous gutter icons provide instant feedback about the entire file on the right-hand side, which is very useful to find errors.

**Comment by keyboardDrummer** [comment]
> However, the continuous gutter icons provide instant feedback about the entire file on the right-hand side, which is very useful to find errors.

Don't errors already show up as red on the minimap?

**Comment by MikaelMayer** [comment]
> Don't errors already show up as red on the minimap?

Maybe, it has been a while since I haven't worked without gutter icons, you might be right!

> Shouldn't the line then not be continuous for the module or class that contains errors?

If the entire file is a module that contains errors then there would be no green at all, which goes against the idea of showing modularly verified functions, right?

**Comment by keyboardDrummer** [comment]
> If the entire file is a module that contains errors then there would be no green at all, which goes against the idea of showing modularly verified functions, right?

I was thinking you would still show green in methods that verify, but not if they're inside a module that does not completely verify.
